scripts/new_bare_repo

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+usage()
+{
+cat << EOF
+usage: new_bare_repo <hutch> <repo name>
+
+Script for automating the tedious process of integrating an existing IOC into our git bare repo scheme
+Run in the top level of your IOC
+
+Ex. new_bare_repo rix new_device
+(Creates a new bare repo in /afs/slac.stanford.edu/g/cd/swe/git/repos/package/epics/ioc/rix/ named "new_device.git")
+
+You also might need to run kinit and aklog before hand!
+
+EOF
+}
+
+if [[ ($1 == "--help") || ($1 == "-h") ]]; then
+   usage
+   exit 0
+
+elif [ $# -lt 2 ]; then
+   echo 'Need to specify the hutch and name of the proposed bare repo' >&2
+   usage
+   exit 1
+
+fi
+
+HUTCH=$1
+REPO=$2
+CWD=$(PWD)
```

Comment:
It's either $PWD (as in the env var) or `$(pwd)` as in "run [pwd](https://man7.org/linux/man-pages/man1/pwd.1.html) and get its output"
```suggestion
CWD=${PWD}
```